frontend/src/modules/displays/ModeBanner.tsx

```diff
@@ -1,3 +1,7 @@
+/**
+ * @summary Component to display Mode text in banner
```

Comment:
What is a banner / what is the banner? Is it the large background text at the bottom edge of certain screens?

Reply:
Yes that's right, Its the banner at bottom edge of certain screens